sdks/android/Mentat/library/src/main/java/org/mozilla/mentat/Mentat.java

```diff
@@ -67,14 +67,15 @@ public void cache(String attribute, CacheDirection direction) {
         switch (direction) {
             case FORWARD:
                 result = JNA.INSTANCE.store_cache_attribute_forward(this.rawPointer, attribute);
+                break;
```

Comment:
I'm going to assume we weren't (aren't?) testing any of this, 'cuz this should fail pretty hard.